kotlin/internal/jvm/android.bzl

```diff
@@ -45,16 +46,47 @@ def _kt_android_artifact(
     # TODO(bazelbuild/rules_kotlin/issues/273): This should be retrieved from a provider.
     base_deps = [_ANDROID_SDK_JAR] + deps
 
-    _android_library(
-        name = base_name,
-        visibility = ["//visibility:private"],
-        exports = base_deps,
-        deps = deps if enable_data_binding else [],
-        enable_data_binding = enable_data_binding,
-        tags = tags,
-        exec_properties = exec_properties,
-        **kwargs
-    )
+    exported_target_labels = [kt_name]
+    if "kt_prune_transitive_deps_incompatible" in tags:
+        # TODO(https://github.com/bazelbuild/rules_kotlin/issues/556): replace with starlark
+        # buildifier: disable=native-android
+        _android_library(
+            name = base_name,
+            resource_files = resource_files,
+            exports = base_deps,
+            deps = deps if enable_data_binding else [],
+            enable_data_binding = enable_data_binding,
+            tags = tags,
+            visibility = ["//visibility:private"],
+            **kwargs
+        )
+        exported_target_labels.append(base_name)
+    elif resource_files:
```

Comment:
This conditional also needs to check if there is a `manifest`, otherwise manifests might get left behind.

Reply:
done